package.json

```diff
@@ -14,6 +14,7 @@
     "test": "vitest"
   },
   "dependencies": {
+    "stylelint-config-recommended": "12.0.0",
```

Comment:
add stylelint-config-recommended, because without this module command "stylelint --fix don't work.